### ISSUE TITLE
Wrong column number when hovering mouse over column name

### ISSUE THREAD

**Issue by manenbu** [rstudio/rstudio#5484]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.2.5001
    OS Version      : Windows 10
    R Version       : 3.6.1

### Steps to reproduce the problem

Run this code:
`foo <- matrix(rnorm(60*60), nrow = 60)`
Then click "foo" to open a data viewer window.
Click the ">" button.

### Describe the problem in detail

When hovering over the column number, it shows the wrong number (offset by 10).

![bug1](https://user-images.githubusercontent.com/7777381/65841666-35c94e80-e368-11e9-8861-dc1cfcf4589d.png)

![bug2](https://user-images.githubusercontent.com/7777381/65841668-39f56c00-e368-11e9-9229-85bdbba0deb0.png)

The hover in the second image shows column 70. There are only 60 columns in the matrix.

### Describe the behavior you expected

I expect it to show the correct column numbers, i.e. 11 in the first image and 60 in the second.

```
R version 3.6.1 (2019-07-05)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 17134)

Matrix products: default

locale:
[1] LC_COLLATE=English_Australia.1252  LC_CTYPE=English_Australia.1252    LC_MONETARY=English_Australia.1252 LC_NUMERIC=C                      
[5] LC_TIME=English_Australia.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
[1] compiler_3.6.1 tools_3.6.1 
```

**Comment by ronblum** [comment]
@manenbu Thank you for reporting the issue! As you can see above, we're targeting fixing this in the next major release of RStudio.

**Comment by dfalty** [comment]
Verified fixed 1.3.822-1
